modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/settings/Settings.es.js

```diff
@@ -34,17 +35,17 @@ export default ({deploymentType, settings = () => <></>, subtitle, title}) => {
 
 	return (
 		<>
-			<div className="autofit-row mb-3 pl-4 pr-4">
-				<div className="autofit-col-expand">
+			<ClayLayout.ContentRow className="mb-3 pl-4 pr-4">
+				<ClayLayout.ContentCol expand>
 					<section className="autofit-section">
```

Comment:
`<ClayLayout.ContentSection containerElement="section">` ? https://github.com/liferay/clay/blob/master/packages/clay-layout/src/Content.tsx#L151

Reply:
oh, great!

Reply:
Can you add it? or do you prefer a PR re-sent?